src/components/AssetSelector.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'styled-components';
+import { OptionProps } from 'react-select';
+
+import { translateRaw } from '@translations';
+import { Asset, ISwapAsset, TSymbol, TUuid } from '@types';
+import { AssetIcon, Typography, Divider, Selector } from '@components';
+
+const SContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 14px 15px 14px 15px;
```

Comment:
Can we use `SPACING`?

Reply:
Same goes for other places.

Reply:
tough one since we don't only have a `10px` and a `20px` value.
If we want to change the styles we can do so in subsequent PR.